test/test_parser.cpp

```diff
@@ -5786,8 +5786,9 @@ TEST(Parser_Geospatial)
     verify_query_sub(test_context, table,
                      "location GEOWITHIN geoPolygon({[$0, $1], [$2, $3], [$4, $5], [$6, $7], [$8, $9]})", coord_args,
                      1);
-    GeoBox b = box.get<GeoBox>();
-    coord_args = {b.lo.longitude, b.lo.latitude, b.hi.longitude, b.hi.latitude};
+    GeoPolygon b = box.get<GeoPolygon>();
```

Comment:
This is quite unexpected to only be able to get polygon back after box. To me it looks like artificially created constraint.

Reply:
Updated!